joplin/urls.py

```diff
@@ -9,14 +9,16 @@
 from wagtail.core import urls as wagtail_urls
 from wagtail.documents import urls as wagtaildocs_urls
 from base.views import joplin_views
+from base.views import joplin_search_views
 from django.urls import reverse
-from base.models import HomePage
+# from base.models import HomePage
```

Comment:
lets remove stuff rather than commenting it out. 

However, I'm wondering if we might still be able to leverage the HomePage model for filtering out on the search view....

Reply:
That was kinda my thinking toward "not" moving it out. I wanted to see y'all perspective on if you wanted it or not. 

The thoughts I had about this were, Our search page is our home page. But, that pages is also a search page... by it's very nature. 

I'm gonna take it out and put that note in.